src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -27,147 +28,86 @@ const styles = {
-    style,
-    label
-  };
-};
-
-const statusCellData = ({status = '- -', darCollectionId, style = {}, label = 'status'}) => {
-  return {
-    data: status,
-    id: darCollectionId,
-    style,
-    label
-  };
-};
-
-const CancelCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `cancel-collection-${collection.id}`,
-    label: 'Cancel',
-    disabled: includes(determineCollectionStatus(collection))(nonCancellableCollectionStatuses),
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const ResubmitCollectionButton = (props) => {
-  const { collection } = props;
-  return h(SimpleButton, {
-    keyProp: `resubmit-collection-${collection.id}`,
-    label: 'Revise',
-    baseColor: Theme.palette.secondary,
-    additionalStyle: {
-      width: '30%',
-      padding: '2%',
-      fontSize: '1.45rem',
-    },
-    onClick: () => props.showConfirmationModal(collection)
-  });
-};
-
-const actionsCellData = ({collection, showConfirmationModal}) => {
-  const { darCollectionId } = collection;
-  const cancel = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'cancel-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `cancel-collection-cell-${darCollectionId}`
-      },
-      [h(CancelCollectionButton, {collection, showConfirmationModal})]
-    )};
-  const revise = {
-    isComponent: true,
-    id: darCollectionId,
-    label: 'resubmit-button',
-    data: div(
-      {
-        style: {
-          display: 'flex',
-          justifyContent: 'left'
-        },
-        key: `resubmit-collection-cell-${darCollectionId}`
-      },
-      [h(ResubmitCollectionButton, {collection, showConfirmationModal})]
-    )
-  };
-
-  return isCollectionCanceled(collection) ? revise : cancel;
+  const {darCode, name, submissionDate, pi, institution, datasetCount, status, actions} = columnHeaderFormat;
+  return [darCode, name, submissionDate, pi, institution, datasetCount, status, actions];
 };
 
 
-const processCollectionRowData = (collections, showConfirmationModal) => {
+const processCollectionRowData = (collections, showConfirmationModal, actionsFlag) => {
   if(!isNil(collections)) {
     return collections.map((collection) => {
-      const { darCollectionId, darCode, createDate } = collection;
+      const { darCollectionId, darCode, createDate, datasets } = collection;
+      const institution = collection.createUser.institution.name;
       /*I want the election-dependent status to be explicit so that the
       researcher knows why they can't cancel the collection*/
       const status = determineCollectionStatus(collection);
       const projectTitle = getProjectTitle(collection);
+      const actionsButton = actionsFlag
```

Comment:
I should have given a better example when first explaining this. I used `actionsFlag` as a spur of the moment example when I should've used something like `actionsDisabled`. 

The reason why is we want to change this component without breaking its implementation in other components. Right now this table is being used in both the SO Console and the NewResearcherConsole. The "flag" is new, which means neither component is passing in a true value, so the flag is currently false. 

With this implementation both the SO Console and the NewResearcherConsole will not show the buttons at all, SO because it's passing in false, NewResearcherConsole because it hasn't been updated. You could update NewResearcherConsole to include this key, but a better option is to just do something like (this is pseudocode)

`actionsDisabled = true (passed in to SO console as a prop); actionsDisabled ? div : button`

This way you only have to update the SO Table and nothing else.